modules/memfault-firmware-sdk/memfault_ncs_metrics.c

```diff
@@ -60,15 +58,7 @@ static void stack_check(const struct k_thread *cthread, void *user_data)
 		return;
 	}
 
-	if (stack_type == STACK_AT_CMD) {
-		LOG_DBG("Unused at_cmd_socket_thread stack size: %d", unused);
-
-		err = memfault_metrics_heartbeat_set_unsigned(
-			MEMFAULT_METRICS_KEY(Ncs_AtCmdUnusedStack), unused);
```

Comment:
The definition of the metric should also be removed from `nrf/modules/memfault-firmware-sdk/config/memfault_metrics_heartbeat_extra.def:4`

Reply:
Thanks, updated